src/coreclr/src/debug/createdump/createdump.h

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#pragma once
```

Comment:
I was pretty sure `#pragma once` was a VS thing.  Don't we want header guards here for Unix too.

Reply:
It works on all major compilers (GCC 3.4, Clang, MSVC), but it's less portable in some ways (a lot of it is implementation specific, some generate a define, some use the path to disambiguate). Createdump is only used here so it should be safe, but the define guards might just be better.

Reply:
I remove the arrayholder.h one and leave the one in createdump.h